addonscript/README.md

```diff
@@ -5,9 +5,10 @@ Addonscript
 
 This is designed to be run from scriptworker, but runs perfectly fine as a standalone script.
 
-
 Docs
 ----
+More details on what this script does can be found in [RelEng docs](http://docs.mozilla-releng.net/en/latest/addons/addons.html).
```

Comment:
I moved this to http://docs.mozilla-releng.net/en/latest/addons/langpacks.html aka https://moz-releng-docs.readthedocs.io/en/latest/addons/langpacks.html in https://github.com/mozilla/build-relengdocs/pull/51  :)

Reply:
Good catch, I've updated it! :)